src/webui/www/private/scripts/client.js

```diff
@@ -672,6 +672,28 @@ window.addEventListener("DOMContentLoaded", () => {
             tracker.classList.toggle("selectedFilter", (Number(tracker.id) === selectedTracker));
     };
 
+    const statusSortOrder = Object.freeze({
```

Comment:
>same sort order as in the GUI but I could use some double check here

Yeah the order is correct. The GUI order is here: https://github.com/qbittorrent/qBittorrent/blob/8e941a06f15a0a8c12f37439cb2815fec73797c3/src/base/bittorrent/torrent.h#L79-L107